### PR TITLE
Added two Smart Bee Design Boards

### PR DESCRIPTION
**Pull request by strid3r21**
I'm adding the files for two boards from my company Smart Bee Designs. its the first time I've made micro-python firmware and therefor first time submitting a PR to this repo for them. so please let me know if I'm missing something or need to make any changes.

thanks

### CODE REVIEW

File: ports/esp32/boards/SBD_BEES3/board.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    ],
+    "docs": "",
+    "features": [
+        "BLE",
+        "WiFi",
+        "USB-C",
+        "RGB LED"
+    ],
+    "images": [
+        "	bees3.jpg"
```
Comment (mattytrentini) [inline ports/esp32/boards/SBD_BEES3/board.json:13]: Are the leading spaces intended?

Comment (strid3r21) [reply ports/esp32/boards/SBD_BEES3/board.json]: no they weren't intentional. I've fixed them.

**Comment by strid3r21** [comment]
@mattytrentini was there anything else I needed to do here to get it merged? You said on Twitter it looked good, but wasn't sure if there was anything else I needed to do.

Thanks!

File: ports/esp32/boards/SBD_BEES3/board.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "deploy": ["../deploy.md"],
+  "docs": "",
+  "features": ["BLE", "WiFi", "USB-C", "RGB LED"],
+  "images": ["bees3.jpg"],
+  "mcu": "esp32s3",
+  "product": "Bee S3",
+  "thumbnail": "",
+  "url": "",
```
Comment (mattytrentini) [inline ports/esp32/boards/SBD_BEES3/board.json:9]: Do you have a URL to link back to where details of the board can be found? Maybe the Tindie page?

**Comment by mattytrentini** [comment]
The board definitions look good but unfortunately I don't have hardware to test them on. 
There doesn't seem to be anything further to do, we just need to let the core devs review and merge this.

**Comment by mattytrentini** [comment]
A couple of other points @strid3r21:
- Code formatting needs to pass (see [Code auto-formatting](https://github.com/micropython/micropython/blob/master/CODECONVENTIONS.md#code-auto-formatting))
  - pre-commit support has been recently added which you might find easier to use
- Commit messages - on _all_ commits - needs to be [in a particular format](https://github.com/micropython/micropython/blob/master/CODECONVENTIONS.md#git-commit-conventions)
  - Ideally you should modify the git history but if you're not comfortable with git just let use know and we can be modify it before merging
- The two `deploy.md`'s are nearly the same; would be best to use just one so it's easier to maintain
  - Or, better, just use the `boards/deploy_s3.md`
  - If you think `deploy_s3.md` could be improved then I'd suggest raising a _separate_ PR
- It would be nice to extract code that's reused
  - The obvious example is `rgb_color_wheel()` which already exists in a few places...though I'm not sure of the best place for it!

**Comment by strid3r21** [comment]
> A couple of other points @strid3r21:
> - Code formatting needs to pass (see [Code auto-formatting](https://github.com/micropython/micropython/blob/master/CODECONVENTIONS.md#code-auto-formatting))
>   - pre-commit support has been recently added which you might find easier to use
> - Commit messages - on _all_ commits - needs to be [in a particular format](https://github.com/micropython/micropython/blob/master/CODECONVENTIONS.md#git-commit-conventions)
>   - Ideally you should modify the git history but if you're not comfortable with git just let use know and we can be modify it before merging
> - The two `deploy.md`'s are nearly the same; would be best to use just one so it's easier to maintain
>   - Or, better, just use the `boards/deploy_s3.md`
>   - If you think `deploy_s3.md` could be improved then I'd suggest raising a _separate_ PR
> - It would be nice to extract code that's reused
>   - The obvious example is `rgb_color_wheel()` which already exists in a few places...though I'm not sure of the best place for it!

Matt, thanks for taking the time to offer some advice on this PR.

However I'm very confused on the latter 2 points. The deploys are for each board and other boards that have already been accepted into the repo are used in the same way. (A deploy for each board. See any of UMs boards)

The rgb_color_wheel() example is in the same boat. There are multiple of them because they're in two different unique helper files for each board. I can't limit it down to one use without making a combined helper file for both boards which I don't want to do. I'd like to keep it as simple as possible for the end user.

This is again something that other boards like UMs boards have done the same way and has already been accepted into the repo.

The commit formatting I understand and can fix.

Let me know what you think and maybe I'm just completely miss understanding what you're asking for in the last 2 points.

**Pull request closed without merging**